src/pages/map/index.tsx

```diff
@@ -24,8 +29,20 @@ const desktopTextLimit = 80;
 const mobileTextLimit = 30;
 
 export default function Page() {
+  const [selectedLandmark, setSelectedLandmark] = useState<Landmark | null>(null);
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
remove this line. you don't need `isOpen` given that you already have `selectedLandmark`.